### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by banlon-jones**
- Created a class Classroom
- Created the has-many/belongs-to relationship between Classroom and Student.
- Created a class Book
- Created a class Rental

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (Emmyn5600) [inline book.rb]: - [ ] As the `attr_accessor` creates both the READER & WRITER methods. I would recommend including the `rentals` in the attr_accessor too.

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```
Comment (Emmyn5600) [inline rental.rb]: - [ ] As the `attr_accessor` creates both the READER & WRITER methods. I would recommend including the `book and person` arguments in the attr_accessor too.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @banlon-jones 

Good job so far! I know you have worked hard on making this milestone. There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes :recycle:
- [ ] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 1 above)
Comment (banlon-jones) [reply book.rb]: Thanks for the feeback 
i have resolved it

File: rental.rb (hunk 2 above)
Comment (banlon-jones) [reply rental.rb]: Thanks for the feeback 
i have resolved it

**Review by Emmyn5600 — APPROVED** [review]
Hi @banlon-jones 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [x] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**